docs/training.rst

```diff
@@ -43,7 +43,8 @@ Training via list data
 For the training, process, you will need to pass in a list of statements where the order of each statement is based
 on it's placement in a given conversation.
 
-For example, if you were to run bot of the following training calls, then the resulting chatterbot would respond to
+For example, if you were to run bot of the following training calls, then the resulting chatterbot would creates 
+a default database in current directory with name ``database.db`` and cahtterbot will respond to
```

Comment:
What if they have specified a different storage adapter? Do you think this is too specific? The training class doesn't just save to a `database.db` file, it saves to whatever storage endpoint is defined by the chat bot's storage adapter. So essentially, the `database.db` file has nothing to do with training, it all depends on the storage adapter settings.